### PR TITLE
Test for edit schedule and details icon

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@benpatterson @muhammad-ammar @kashifch 
Please review.

File: regression/pages/studio/course_outline_page.py [hunk 1]
```diff
@@ -98,3 +99,12 @@ def delete_section(self):
         ).first.click()
         self.wait_for_ajax()
         click_confirmation_prompt_primary_button(self)
+
+    def click_edit_start_date_button(self):
+        """
+        Clicks edit start date button on Course Outline page
+        """
+        button = self.q(css='.action-button span.icon.fa.fa-pencil').results[0]
+        ActionChains(
```
Comment (benpatterson) [inline regression/pages/studio/course_outline_page.py]: Could you add a comment describing why you need an `ActionChain` to perform the click?

File: regression/pages/studio/schedule_and_details_page.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+"""
+Schedule and Details Page for Studio
+"""
+from bok_choy.page_object import PageObject
+from regression.pages.studio import BASE_URL
+
+
+class StudioScheduleDetails(PageObject):
+    """
+    Schedule and Details Page for Studio
```
Comment (benpatterson) [inline regression/pages/studio/schedule_and_details_page.py]: nit: Point out this is for course details + schedule

File: regression/tests/studio/test_course_outline.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+        self.studio_course_outline = CourseOutlinePageExtended(
+            self.browser, self.course_info['org'], self.course_info['number'],
+            self.course_info['run'])
+
+        self.studio_home_page.visit()
+        self.studio_home_page.select_course(get_course_display_name())
+        self.studio_course_outline.wait_for_page()
+
+    def test_edit_start_button(self):
```
Comment (benpatterson) [inline regression/tests/studio/test_course_outline.py]: Do you also want a test where you change the date & see that it's been saved?

File: regression/pages/studio/course_outline_page.py (hunk 1 above)
Comment (raeeschachar) [inline regression/pages/studio/course_outline_page.py]: Done

File: regression/pages/studio/schedule_and_details_page.py (hunk 2 above)
Comment (raeeschachar) [inline regression/pages/studio/schedule_and_details_page.py]: Done

File: regression/tests/studio/test_course_outline.py (hunk 3 above)
Comment (raeeschachar) [inline regression/tests/studio/test_course_outline.py]: @benpatterson 
No, that is another ticket. This one was to test this button only.

**Comment by raeeschachar** [comment]
@benpatterson 
Comments addressed. Please review.

**Comment by benpatterson** [comment]
👍

**Pull request merged**